### DISCUSSION TITLE
Visualize the generated log.json file

### DISCUSSION THREAD

**Discussion by Emily0819** [open-mmlab/mmgeneration#456] [How-to] [answered]
How to visualize the generated log.json file? Is there a special .py file?

**Comment by plyfager** [comment, marked as answer]
Hi @Emily0819, we hasn't provided a script for visualize `json` file. However, you can use visualization backend like tensorboard for your purpose. Just install tensorboard and add lines below.
```
log_config = dict( 
    interval=100, 
    hooks=[ 
        dict(type='TextLoggerHook'), 
        dict(type='TensorboardLoggerHook') 
    ]) 
```
Then train your model with commands below.
```
bash tools/dist_train.sh xxx
tensorboard --logdir work_dirs/xxx 
```

For other choices, you can refer to [https://zhuanlan.zhihu.com/p/387078211](url) 

There may be some occasions that visualization backend cannot be used and a script would be better, we will appreciate it a lot if you would like to offer one.

**Comment by Emily0819** [comment]
@plyfager Thank you for your reply!
I trained stylegan2 with my own dataset. What I set is b4*2 and the source code is b4*8. Is it necessary to divide the initial learning rate by 4 in this case?
